jb install fails with 0.7

<!--

Feel free to ask questions in #prometheus-operator on Kubernetes Slack!

-->

**What happened?**

the command `jb install github.com/coreos/kube-prometheus/jsonnet/kube-prometheus@release-0.7`

generate a panic on rename

**How to reproduce it (as minimally and precisely as possible)**:

```
jb init
jb install github.com/coreos/kube-prometheus/jsonnet/kube-prometheus@release-0.7
```

error : 
```
GET https://github.com/coreos/kube-prometheus/archive/4dbc23c923a8c1978e7daea6eb301623d75a9be1.tar.gz 200
GET https://github.com/kubernetes/kube-state-metrics/archive/e72315512a38653b19dcfe4429f93eadedc0ea96.tar.gz 200
GET https://github.com/prometheus/prometheus/archive/26d89b4b0776fe4cd5a3656dfa520f119a375273.tar.gz 200
GET https://github.com/prometheus/alertmanager/archive/193ebba04d1e70d971047e983a0b489112610460.tar.gz 200
GET https://github.com/etcd-io/etcd/archive/e8ba375032e8e48d009759dfb285f7812e7bcb8c.tar.gz 200
panic: rename /Users/hleclerc/tmp/jjj/vendor/.tmp/jsonnetpkg-github.com-etcd-io-etcd-contrib-mixin-e8ba375032e8e48d009759dfb285f7812e7bcb8c647930610/contrib/mixin /Users/hleclerc/tmp/jjj/vendor/github.com/etcd-io/etcd/contrib/mixin: no such file or directory

goroutine 1 [running]:
github.com/jsonnet-bundler/jsonnet-bundler/pkg.(*GitPackage).Install(0xc0000b20f0, 0x14ca7a0, 0xc0000b6030, 0xc0001e4e70, 0x25, 0xc0000b85e0, 0x1e, 0xc00044de30, 0x28, 0x0, ...)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/git.go:220 +0x1d56
github.com/jsonnet-bundler/jsonnet-bundler/pkg.download(0xc00051c410, 0x0, 0xc00044de30, 0x28, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc0000b85e0, ...)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:289 +0x1aa
github.com/jsonnet-bundler/jsonnet-bundler/pkg.ensure(0xc0004efe90, 0xc0000b85e0, 0x1e, 0xc00009bf80, 0x0, 0x0, 0x0)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:233 +0x6c8
github.com/jsonnet-bundler/jsonnet-bundler/pkg.ensure(0xc00009bf20, 0xc0000b85e0, 0x1e, 0xc00009bf80, 0x1e, 0x0, 0x1)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:259 +0xef6
github.com/jsonnet-bundler/jsonnet-bundler/pkg.Ensure(0xc00009bf20, 0x1, 0xc0000b85e0, 0x1e, 0xc00009bf80, 0x0, 0x0, 0x0)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:55 +0x67
main.installCommand(0xc00001a044, 0x17, 0x14352ca, 0x6, 0xc000098fc0, 0x1, 0x1, 0x0, 0x0)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb/install.go:75 +0x74c
main.Main(0xc00008e058)
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb/main.go:82 +0xb07
main.main()
	/Users/hleclerc/go/src/github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb/main.go:37 +0x22
	
```

--- a/jsonnet/kube-prometheus/jsonnetfile.json
+++ b/jsonnet/kube-prometheus/jsonnetfile.json
@@ -17,7 +17,7 @@
           "subdir": "contrib/mixin"
         }
       },
-      "version": "e8ba375032e8e48d009759dfb285f7812e7bcb8c"
+      "version": "60d5159091ab06e80ad446ce9e4f415e5f53439e"
     },
     {
       "source": {

